sdcard/os/archlinux.sh

```diff
@@ -21,6 +23,8 @@ initos(){
             generaldownload;;
         cubietruck|bananapro)
             allwinnerdownload;;
+        odroid-c2)
```

Comment:
tried `curl -sSL -k http://archlinuxarm.org/os/ArchLinuxARM-odroid-c2-latest.tar.gz | tar -xz -C odroid-c2` without problems on an archlinux host. Works without problems

Of course there are the `tar: Ignoring unknown extended header keyword 'SCHILY.fflags'` warnings, but one may ignore them